packages/google-types/src/version/index.ts

```diff
@@ -27,6 +27,7 @@ export interface BaseGoogleVersionData extends Version.VoiceVersionData<Voice> {
   status: { stage: GoogleStage };
   settings: BaseGoogleVersionSettings;
   publishing: BaseGoogleVersionPublishing;
+  modelVersion: number;
```

Comment:
if you type this as
```suggestion
  modelVersion: 1;
```
it'll work as a discriminated union